### PR TITLE
SDT-128: SDT Commissioning - convert to annotation based config

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SDT-128

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: handlers/build.gradle [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    testClassesDirs = sourceSets.testUnit.output.classesDirs
+    classpath = sourceSets.testUnit.runtimeClasspath
+    failFast = true
+}
+
+check.dependsOn unit
+
+test {
+    useJUnitPlatform()
+}
```
Comment (paulridout) [inline handlers/build.gradle:62]: Is this needed?  It's not present in the transformers or validators build.gradle files.  If it is needed then add it to the transformers and validators build.gradle files.  If it is not needed then remove it from here.

File: transformers/build.gradle [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+tasks.register('unit', Test) {
+    description = "Runs unit tests"
+    group = "Verification"
+    testClassesDirs = sourceSets.testUnit.output.classesDirs
+    classpath = sourceSets.testUnit.runtimeClasspath
+    failFast = true
+}
+
+check.dependsOn unit
```
Comment (paulridout) [inline transformers/build.gradle:56]: Does this need`test {useJunitPlatform()}` (with the right indentation) here?  See comment for handlers build.gradle file.

File: validators/build.gradle [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+tasks.register('unit', Test) {
+    description = "Runs unit tests"
+    group = "Verification"
+    testClassesDirs = sourceSets.testUnit.output.classesDirs
+    classpath = sourceSets.testUnit.runtimeClasspath
+    failFast = true
+}
+
+check.dependsOn unit
```
Comment (paulridout) [inline validators/build.gradle:56]: Does this need`test {useJunitPlatform()}` (with the right indentation) here?  See comment for handlers build.gradle file.

File: handlers/build.gradle (hunk 1 above)
Comment (kiran-yenigala-hmcts) [reply handlers/build.gradle:62]: removed it from handlers

File: transformers/build.gradle (hunk 2 above)
Comment (kiran-yenigala-hmcts) [reply transformers/build.gradle:56]: not needed

File: validators/build.gradle (hunk 3 above)
Comment (kiran-yenigala-hmcts) [reply validators/build.gradle:56]: not needed

**Review by paulridout — APPROVED** [review]
Changes look OK

**Pull request merged**